site/content/blog/2021-05-01-whats-new-in-svelte-may-2021.md

```diff
@@ -0,0 +1,74 @@
+---
+title: What's new in Svelte: May 2021
+description: Working toward SvelteKit 1.0 and a showcase full of SvelteKit sites!
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Last week, Svelte Summit blew us away with a mountain of content! [Check out the full recording](https://www.youtube.com/watch?v=fnr9XWvjJHw) or an audio-only (p)review [on Svelte Radio](https://www.svelteradio.com/episodes/svelte-summit-party-episode). Now lets get into this month's news...
+
+## Svelte Compiler Updates
+- No new releases this month? To be deleted if these changes stay "Unreleased": https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#unreleased
+
+## New in SvelteKit
+- The new [SvelteKit Demo Site](https://netlify.demo.svelte.dev/) looks better than ever after running `npm init svelte`
```

Comment:
kit.svelte.dev was reimplemented and got an awesome new homepage logo. The code powering it lives at [sveltejs/sites](https://github.com/sveltejs/sites) and we hope to move the main svelte.dev site there in the future as well